Docsp 15085 aggregations in updates (#192)
mongodb/docs-node: source/fundamentals/crud/write-operations/change-a-document.txt

The update operators apply only to the fields associated with them in your
update document.

.. note:: Aggregation Pipelines in Update Operations

   If you are using MongoDB Version 4.2 or later, you can use aggregation
   pipelines made up of a subset of aggregation stages in update operations. For

>    more information, including what aggregation stages MongoDB supports in

Reviewer: ```suggestion
   more information on the aggregation stages MongoDB supports in
```
Suggestion: There doesn't seem to be any other information you can be referencing, so saying "including" was confusing for me.

Author: Thanks for the note. I agree that ", including what" can be replaced by "on the".  
Committing suggested change. 

I agree that "for more information, including..." is not very helpful (more information be be clarified right after it is mentioned).

>    aggregation pipelines used in update operations, see this tutorial on building

Reviewer: ```suggestion
   aggregation pipelines used in update operations, see our tutorial on building
```
Nit: I like to know who's content/material a link belongs to before considering to view it.

Author: Thanks for the note. I agree that knowing who is hosting a link or tutorial is helpful information for readers. Committing suggested change.

Perhaps this is something we could codify in a team PR review? (I.E. use "our" when mentioning content produced by MongoDB)
